pkg/dispatchcli/cmd/manage_bootstrap.go

```diff
@@ -352,9 +359,9 @@ func runBootstrap(out, errOut io.Writer, cmd *cobra.Command, args []string) erro
 	policyName := defaultPolicyName
 	fmt.Fprintf(out, "Creating Policy: %s\n", policyName)
 	// Deleting any existing policy
-	_, err = iamClient.DeletePolicy(context.TODO(), policyName)
+	_, err = iamClient.DeletePolicy(context.TODO(), bootstrapOrg, policyName)
 	if err != nil {
-		if _, ok := err.(*client.ErrorNotFound); !ok {
+		if _, ok := err.(*policy.DeletePolicyNotFound); !ok {
```

Comment:
we should be using `client` errors

Reply:
Yup, the result of a bad rebase, I fixed it after testing.